aws/vpc-public-only/README.md

```diff
@@ -0,0 +1,38 @@
+# aws/vpc-public-only - Virtual Private Cloud
+This module is used to create a VPC along with the necessary configuration to
+be useful.
+
+## What this does
+
+ - Create VPC named after `app_name` and `app_env`
+ - Create public subnet for each `aws_zones` specified
+ - Provision a Internet Gateway and configure public subnets to route through it
+ - Create a DB subnet group including all public subnets
+
+## Required Inputs
+
+ - `app_name` - Name of application, ex: Doorman, IdP, etc.
+ - `app_env` - Name of environment, ex: production, testing, etc.
+ - `aws_zones` - A list of zones to create subnets in, ex: `["us-east-1c", "us-east-1d", "us-east-1e"]`
+
+## Optional Inputs
+
+ - `enable_dns_hostnames` - default `false`
```

Comment:
The actual default value for this `enable_dns_hostnames` input (in the `vars.tf` file) is `true`, not `false`.

Reply:
thanks